extra/fuegogin/adaptor.go

```diff
@@ -9,6 +9,24 @@ import (
 	"github.com/go-fuego/fuego/internal"
 )
 
+type OpenAPIHandler struct {
+	*gin.Engine
```

Comment:
Please do not embed this field. We don't need embedding here!

Reply:
Do you care for a name? “Handler” maybe. I really don’t have an idea

Reply:
`GinEngine` ?

Reply:
Updated